### DISCUSSION TITLE
Application dashboard / aggregating all URLs

### DISCUSSION THREAD

**Discussion by linus-ha** [louislam/dockge#322] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

A customizable dashboard view, aggregating all URLs of stacks. Similar to https://github.com/linuxserver/Heimdall, but integrated into dockge. Should be able to publish this dashboard, similar to how Uptime Kuma handles this.

### ✔️ Solution

Dashboard functionality as seen in Uptime Kuma, aggregating URLs from stack compose files.

**Comment by sharkoz** [comment]
That would be awesome ! And make the 'url' option really useful. Otherwise as of now it is not really put in good use.
By default it could use the host:port link already available in the pill button under the container name

**Comment by ak127a** [comment]
+1
Yess!!  Would be so nice to have just 1 dashboard to go to all hosted stacks, hard to remember so many port numbers! :grin:
